L/libdmg_hfsplus/build_tarballs.jl

```diff
@@ -0,0 +1,69 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libdmg_hfsplus"
+version = v"0.0.0"
```

Comment:
That doesn't sound like a meaningful version number

Reply:
I don't know what to do here. It progressed because the library `libdmg-hfsplus` was released in an alpha state and never tagged. Then Mozilla figured out that it could be used for Firefox distribution, forked the repo, and added some additional maintenance work. They also did not tag it. 

Perhaps I may write to the Mozilla fork maintainers to ask if they could tag the version

Reply:
I would suggest going with 0.0.YYYYMMDD for untagged/unreleased projects - so no real version numbers (below 0.1.0) is interfered with.

Reply:
I sent an email to the maintainer. Perhaps he could tag a release on his side.

Reply:
Solved https://github.com/mozilla/libdmg-hfsplus/releases/tag/0.5